test/src/main/java/org/axonframework/test/matchers/Matchers.java

```diff
@@ -155,6 +155,17 @@ public static Matcher<List<CommandMessage<?>>> noCommands() {
         return new EmptyCollectionMatcher<>("commands");
     }
 
+    /**
+     * Returns a Matcher that matches with exact class type defined by the given {@code expected}.
+     *
+     * @param expected The expected event class
+     * @param <T>      The type of event to match against
```

Comment:
This `Matcher` is not restricted to events :-)

```suggestion
     * @param expected The expected class
     * @param <T>      The object type to match the given {@code expected} class with.
```

Reply:
JavaDoc updated